packages/core/src/RenderingEngine/vtkClasses/vtkStreamingOpenGLVolumeMapper.js

```diff
@@ -418,28 +416,47 @@ function vtkStreamingOpenGLVolumeMapper(publicAPI, model) {
 
         program.setUniformMatrix('vWCtoIDX', worldToIndex);
 
+        const camera = ren.getActiveCamera();
+        const [cRange0, cRange1] = camera.getClippingRange();
+        const distance = camera.getDistance();
+
+        // set the clipping range to be model.distance and model.distance + 0.1
+        // since we use the in the keyMats.wcpc (world to projection) matrix
```

Comment:
I think 'since we use the in the keyMats.wcpp...' should be 'since we use **this** in the keyMats.wcpp...'